frontend/views/containers/access/SignupForm.vue

```diff
@@ -62,27 +61,24 @@ export default {
   components: {
     ModalTemplate,
     PasswordForm,
-    BannerScoped
+    BannerScoped,
+    ButtonSubmit
   },
   data () {
     return {
       form: {
         name: null,
         password: null,
         email: null
-      },
-      ephemeral: {
-        isSubmitting: false
       }
     }
   },
   methods: {
     async signup () {
-      // Prevent autocomplete submission when empty field
-      if (!this.form.name || !this.form.email || !this.form.password || this.ephemeral.isSubmitting) {
+      if (this.$v.form.$invalid) {
+        this.$refs.formMsg.danger('The fields are incompleted.')
```

Comment:
Same two concerns as above in LoginForm (see comment there)

Reply:
I think this was missed. Still needs an `L` function.

While we're updating this, can you please update the copy too to read: `L('The form is invalid.')`?